src/components/StudioTool.tsx

```diff
@@ -0,0 +1,25 @@
+import {studioTheme, ThemeProvider} from '@sanity/ui'
+import React from 'react'
+import {Tool} from 'sanity'
+
+import {Config} from '../util/types'
+import ToolIcon from './icons/ToolIcon'
+import VideosBrowser from './VideosBrowser'
+
+const StudioTool: React.FC<Config> = () => {
+  return (
+    <ThemeProvider theme={studioTheme}>
```

Comment:
Good catch! I added this in because I was initially using the `npm link` approach that forks the versions of `sanity` between the plugin and the studio code. Removing it ✅